### PR TITLE
Temporarily exclude unstable connectors from catalog

### PR DESCRIPTION
**Pull request by lburgazzoli**

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

**Review by oscerd — APPROVED** [review]

File: cos-fleet-catalog-tools/camel-connector-maven-plugin/src/main/java/org/bf2/cos/catalog/camel/maven/connector/GenerateAppMojo.java [hunk 1]
```diff
@@ -99,81 +101,85 @@ public void execute() throws MojoExecutionException, MojoFailureException {
             this.indexFile = indexPath.toPath().resolve("connectors.json");
             this.manifestLocalFile = definitionPathLocal.toPath().resolve(this.manifestId + ".json");
 
-            if (!Files.exists(manifestLocalFile)) {
+            if (!Files.exists(manifestLocalFile) && !exclude) {
                 getLog().warn("Skipping App build as the definition file " + manifestLocalFile.getFileName() + " is missing");
```
Comment (rinaldodev) [inline cos-fleet-catalog-tools/camel-connector-maven-plugin/src/main/java/org/bf2/cos/catalog/camel/maven/connector/GenerateAppMojo.java:105]: message is now misleading

File: cos-fleet-catalog-connectors-it-support/src/main/groovy/org/bf2/cos/connector/camel/it/support/TestUtils.groovy [hunk 2]
```diff
@@ -2,7 +2,27 @@ package org.bf2.cos.connector.camel.it.support
...
 import groovy.json.JsonSlurper
 
+import java.nio.file.Files
+import java.nio.file.Path
+
 class TestUtils {
 
     public static final JsonSlurper SLURPER = new JsonSlurper()
+
+    static boolean definitionExists(String definition) {
```
Comment (rinaldodev) [inline cos-fleet-catalog-connectors-it-support/src/main/groovy/org/bf2/cos/connector/camel/it/support/TestUtils.groovy:12]: is this being used?

Comment (lburgazzoli) [reply cos-fleet-catalog-connectors-it-support/src/main/groovy/org/bf2/cos/connector/camel/it/support/TestUtils.groovy]: removed

File: cos-fleet-catalog-tools/camel-connector-maven-plugin/src/main/java/org/bf2/cos/catalog/camel/maven/connector/GenerateAppMojo.java (hunk 1 above)
Comment (lburgazzoli) [reply cos-fleet-catalog-tools/camel-connector-maven-plugin/src/main/java/org/bf2/cos/catalog/camel/maven/connector/GenerateAppMojo.java:105]: This is to retain the old behavior and message when the connector is **not** marked for exclusion.

Added a new log message to log the exclusion: https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/pull/569/files#diff-2645cfc5dc6af74108a4a5c8ae14480e5818d63285a0d605ce2cd2bccf32226cR182

**Review by rinaldodev — APPROVED** [review]

**Pull request merged**